hws/src/dr_rule.py

```diff
@@ -131,7 +131,7 @@ def dr_parse_rules(matcher, verbosity, tabs):
             while ste != None:
                 rule.add_ste(ste)
                 hit_loc = ste.get_hit_location()
-                ste = dr_hw_get_ste_from_loc(hit_loc, hint_loc, False, match_ste_id)
+                ste = dr_hw_get_ste_from_loc(hit_loc, hint_loc, True, match_ste_id)
```

Comment:
let's keep it false, so we still have the optimization, and add the actions STE's to the hint location at the end of the array in  dr_hw_get_ste_from_loc...